plugins/copr.py

```diff
@@ -471,6 +471,10 @@ def _guess_chroot(self):
                 chroot = ("opensuse-leap-{0}-{1}".format(dist[1], distarch))
         elif "CentOS Stream" in dist:
             chroot = ("centos-stream-{0}-{1}".format(dist[1], distarch))
+        elif "Red Hat Enterprise Linux" in dist:
+            chroot = ("epel-%s-x86_64" % dist[1].split(".", 1)[0])
```

Comment:
I'm curuios why to handle RHEL explicitly here, wouldn't the default `else` branch work?

Reply:
> I'm curuios why to handle RHEL explicitly here, wouldn't the default `else` branch work?

RHEL has Fedora in it's ID_LIKE, so RHEL would get detected as a Fedora remix. However this isn't an issue for RHEL alternatives like Rocky/Alma/etc, because those all contain ID_LIKE=rhel or ID_LIKE=centos which the patch checks for.

Reply:
That means that those derivatives have messed up, because they are supposed to have `fedora` in their `ID_LIKE` fields.